eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -4,10 +4,10 @@ Updates package README.md for publishing to docs.microsoft.com
 
 .DESCRIPTION
 Given a PackageInfo .json file, format the package README.md file with metadata
-and other information needed to release reference docs: 
```

Comment:
Are these changes just clean-up?

Reply:
Yeah, I had this file open and my editor removed trailing spaces. I can remove it from the commits if it makes it easier.